app/services/force/request.rb

```diff
@@ -20,11 +20,18 @@ def get(endpoint, params = nil)
       send(:get, endpoint, params)
     end
 
+    def env_variable_true(variable)
+      variable.to_s.casecmp('true').zero?
+    end
+
     def cached_get(endpoint, params = nil, force = false)
       key = "#{endpoint}#{params ? '?' + params.to_query : ''}"
       force = ActiveModel::Type::Boolean.new.cast(force)
-      force_refresh = force || !ENV['CACHE_SALESFORCE_REQUESTS']
-      if ENV['FREEZE_SALESFORCE_CACHE']
+      if force == true
+        puts 'Force is true'
```

Comment:
Was this line intentionally kept in the PR?

Reply:
Yeah I think more log messages would be helpful. I can add some more robust logging.